Pod/Classes/PINRemoteImageCategoryManager.m

```diff
@@ -195,7 +195,13 @@ + (void)setImageOnView:(id <PINRemoteImageCategory>)view
                     return;
                 }
                 if (result.image) {
-                    [view pin_updateUIWithImage:result.image animatedImage:nil];
+                    if ([view respondsToSelector:@selector(pin_updateUIWithRemoteImageManagerResult:)]) {
```

Comment:
Ah, you mean so it doesn't make the same check twice in the same method? Yeah good idea I will make the change.